week0001/fizzbuzz_daudau.php

```diff
@@ -0,0 +1,8 @@
+<?php
+	$i = 1;
+	$fizz = 'fizz'.'<br/>';
+	$buzz = 'buzz'.'<br/>';
+	$a = 3; $b =5;
+	$ab = $a*$b;
+	while ($i++ < 100)
+		echo $i%$ab ? $i%$a ? $i%$b ? false : $buzz : $fizz : $fizz.$buzz;
```

Comment:
Spacing around `%`.

You are over-complicating a super simple problem. Write it in such a way so other people can read it, otherwise they will remove it. Use normal `if/else` and `{}`